reana_commons/workspaces.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2021 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+"""REANA-Commons workspaces util."""
+
+from reana_commons.errors import REANAValidationError
+from reana_commons.config import WORKSPACE_PATHS
+import os
+
+
+def validate_workspace(workspace_option, available_paths=WORKSPACE_PATHS):
+    """Validate and return workspace.
+
+    :param workspace_option: A string of the workspace.
+    :returns: A string of the validated workspace.
+    """
+    if workspace_option:
+        available = any(
+            workspace_option.startswith(os.path.abspath(path))
```

Comment:
Minor: I guess `workspace_option` could be also wrapped with `os.path.abspath()` to catch cases where users are trying to access non allowed locations.
Example: `workspace_option="/eos/../otherdir"`